src/main/java/com/xpandit/plugins/xrayjenkins/services/enviromentvariables/util/XrayEnvironmentVariableSetterHelperUtil.java

```diff
@@ -0,0 +1,264 @@
+package com.xpandit.plugins.xrayjenkins.services.enviromentvariables.util;
+
+import com.google.common.collect.Sets;
+import com.xpandit.plugins.xrayjenkins.model.HostingType;
+import com.xpandit.xray.model.UploadResult;
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class XrayEnvironmentVariableSetterHelperUtil {
```

Comment:
- Javadoc explaining the goal of this class

- We could just call it XrayEnvironmentVariableSetterUtil or XrayEnvironmentVariableSetterHelper